src/main/scala/temple/TempleConfig.scala

```diff
@@ -16,3 +21,7 @@ class TempleConfig(arguments: Seq[String]) extends ScallopConf(arguments) {
   addSubcommand(generate)
   verify()
 }
+
+object TempleConfig {
+  class UnhandledArgumentException extends RuntimeException
```

Comment:
Where is this used?

Reply:
https://github.com/TempleEight/temple/pull/14/files#diff-ce75755896e0616e04f7efac42d83fa9R12